core-bundle/contao/pages/PageRegular.php

```diff
@@ -621,7 +618,14 @@ protected function createFooterScripts($objPage, $objLayout)
 		// Add the custom JavaScript
 		if ($objLayout->script)
 		{
-			$strScripts .= "\n" . trim($objLayout->script) . "\n";
+			$customScript = trim($objLayout->script);
+
+			if ($nonce = $this->Template->nonce('script-src'))
+			{
+				$customScript = str_replace('<script', '<script nonce="' . $nonce . '"', $customScript);
```

Comment:
Meeeeeh. Can we add a comment here that this is done here because we know `$objLayout->script` is a safe user input?

Reply:
Done in c727221965